### ISSUE TITLE
Loading FX preset with deactivated param doesn't work when changing FX types

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#4176]
After #4175 is merged:

1. Open Surge
2. Open EQ3 and deactivate one band
3. Save FX preset

Then:

1. Reopen Surge
2. From no FX loaded, load your saved EQ FX preset -> band will not be deactivated
3. Load the same FX preset again -> band will be deactivated as it should

FX lifecycle shenanigans, apparently.

**Comment by mynameismuhl** [comment]
This fixed it for me but is tested on linux only

**Comment by baconpaul** [comment]
Yeah that fix would break other things alas. The right fix is quite a bit trickier and invokes making sure fx presets are consistently applied after the new fix init ctrltypes are called.

**Comment by baconpaul** [comment]
So ignore me completely. Your fix is right and I misread your diff :( Getting coffee now

https://github.com/surge-synthesizer/surge/blob/main/doc/FXLifecycle.md

There’s the foll documentation on the fx lifecycle. As you can see when you load from snapshot we apply values then call init_ctrltypes but not init_defaultvalues.

You are exactly correct that moving the deactivated state from init_ctryltypes into init_defaultvalues fixes EQ. And I misread your diff as only doing half of that!! (Sorry again)

So the way to close this issue is simply to go through all the REST of the FX and make sure that when we default deactivated we do it in `init_defaultvalues` and not `init_ctryltypes` like your diff does.

I’ll merge your diff when CI is done. @mkruselj if you wanna scan the rest of the FX for this that’d be awesome or I can do it when I’m back!

**Comment by mynameismuhl** [comment]
Cool. Happy this worked.
I can also apply that to the rest of the fx where needed
if you are not doing that already @mkruselj ?

**Comment by baconpaul** [comment]
Thanks! And again sorry for th initial misread!

**Comment by mkruselj** [comment]
> 
> 
> Cool. Happy this worked.
> I can also apply that to the rest of the fx where needed
> if you are not doing that already @mkruselj ?

Feel free to do the rest of FX (and add yourself to AUTHORS file while at it, as mentioned in your PR) :slightly_smiling_face:
